public/app/views/shared/_record_innards.html.erb

```diff
@@ -81,7 +81,9 @@
                                                            :p_id => 'classifications_list', :p_body => x} %>
   <% end %>
 	<% unless @result.agents.blank? %>
```

Comment:
Indentation seems to be off here, but that's true of other portions of the file so it can be addressed separately.